source/extensions/common/wasm/ext/envoy_proxy_wasm_api.h

```diff
@@ -25,6 +26,22 @@ struct DnsResult {
   std::string address;
 };
 
+struct CounterResult {
+  uint64_t delta;
+  std::string name;
```

Comment:
Use StringView here and just alias the string in the buffer.

Reply:
converted to stringView